playbooks/ansible/roles/sit.cephfs/tasks/git/configure.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Configure with Ceph components
+  command: >-
+    ./configure --enable-debug \
+                --prefix=/usr \
+                --enable-fhs \
+                --sysconfdir=/etc \
+                --libdir=/usr/lib64 \
+                --localstatedir=/var \
+                --sharedstatedir=/var/lib \
+                --with-piddir=/run \
+                --with-sockets-dir=/run/samba \
+                --with-lockdir=/var/lib/samba/lock \
+                --with-statedir=/var/lib/samba \
+                --with-cachedir=/var/lib/samba \
+                --disable-rpath-install \
+                --with-quotas \
+                --bundled-libraries=!zlib,!popt,talloc,pytalloc,pytalloc-util,tevent,pytevent,tdb,pytdb,ldb,pyldb,pyldb-util \
```

Comment:
Nit: All the files containig libraries=!zlib,!popt,talloc,pytalloc,pytalloc-util,tevent,pytevent,tdb,pytdb,ldb,pyldb,pyldb-util \
have a yamllint warning of line too long, should we consider fixing it?

Reply:
Sure, but unless we end up refreshing the patches to address other comments(if any) let's ignore the warning for now.